ff/serving/provider/offline.go

```diff
@@ -235,6 +241,14 @@ func (store *memoryOfflineStore) GetMaterialization(id MaterializationID) (Mater
 	return mat, nil
 }
 
+func (store *memoryOfflineStore) DeleteMaterialization(id MaterializationID) error {
+	if _, has := store.materializations[id]; !has {
```

Comment:
Can you add a test for this, easy coverage points

Reply:
added